bindings/tpython/inc/TPython.h

```diff
@@ -67,6 +69,17 @@ class TPython {
    // void* to CPPInstance conversion, returns a new reference
    static PyObject *CPPInstance_FromVoidPtr(void *addr, const char *classname, Bool_t python_owns = kFALSE);
 
+   struct TPyResult {
```

Comment:
Would it be to cumbersome on the Python side if this was an std::variant?

Reply:
I can try, but I suspect it will cause some trouble :slightly_smiling_face:

Reply:
That went in the good direction! But I realized `std::variant` was a bit restrictive, because it's very strict with the type. E.g. if you have a `std::variant<void*, ....>`, you can't assign it with a TObject or so in Python without an explicit cast to `void*`. I have now implemented a solution with `std::any` that works quite well. What do you think?